src/test/java/com/jetbulb/interviews/NotationConverterTest.java

```diff
@@ -0,0 +1,50 @@
+package com.jetbulb.interviews;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowableOfType;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+public class NotationConverterTest {
+    private final NotationConverter converter = new NotationConverter();
```

Comment:
What do you think about generalising this property by using power of OOP principles?